metrics-proxy/pom.xml

```diff
@@ -132,6 +132,11 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.velocity</groupId>
+      <artifactId>velocity</artifactId>
```

Comment:
Please add an exclude for this artifact in config-model/pom.xml's dependency on metrics-proxy. It adds an extra 0.5 MB.